packages/react/package.json

```diff
@@ -29,6 +31,9 @@
     "loose-envify": "^1.1.0",
     "object-assign": "^4.1.1"
   },
+  "peerDependencies": {
```

Comment:
A package can't be its own peer. This isn't needed. If our scripts complain it's our scripts that should be fixed.